python/paddle/fluid/tests/unittests/collective/process_group_gloo.py

```diff
@@ -100,6 +100,22 @@ def test_create_process_group_gloo(self):
             assert np.array_equal(broadcast_result, tensor_y)
         print("test broadcast api ok")
 
+        # test send_recv
+        # rank 0
+        x = np.random.random(self.shape).astype(self.dtype)
+        tensor_x = paddle.to_tensor(x)
+        # rank 1
+        y = np.random.random(self.shape).astype(self.dtype)
+        tensor_y = paddle.to_tensor(y)
+
+        send_recv_result = paddle.assign(tensor_x)
+        if rank == 0:
```

Comment:
* Use `pg.rank` would be better
* add send from 1 to 0

Reply:
Done